### PR TITLE
XW-821 | don't use deprecated functionality in article-comments

### PR DESCRIPTION
**Pull request by hakubo**

### CODE REVIEW

File: front/main/app/components/article-comments.js [hunk 1]
```diff
@@ -67,14 +67,32 @@ export default Ember.Component.extend({
...
+
 		this.set('model', ArticleCommentsModel.create({
 			articleId: this.get('articleId')
 		}));
+	},
+
+	/**
+	 * @returns {void}
+	 */
+	didRender() {
```
Comment (rogatty) [inline front/main/app/components/article-comments.js]: I think we should use `this._super(...arguments);` in all hooks.

File: front/main/app/components/article-comments.js [hunk 2]
```diff
@@ -67,14 +67,32 @@ export default Ember.Component.extend({
 	}),
 
 	/**
+	 * Sets model when we get new articleId
+	 *
 	 * @returns {void}
 	 */
-	didInsertElement() {
+	didInitAttrs() {
```
Comment (rogatty) [inline front/main/app/components/article-comments.js]: Are you sure this shouldn't be `didReceiveAttrs`?

https://guides.emberjs.com/v2.3.0/components/the-component-lifecycle/#toc_formatting-component-attributes-with-code-didreceiveattrs-code

Comment (hakubo) [inline front/main/app/components/article-comments.js]: yes, the way article-comments are build now
creating model can happen only once
when articleId changes there is different hook that will change that

this whole component should be rewritten not to use observers

**Comment by rogatty** [comment]
LGTM

**Pull request merged**